### ISSUE TITLE
[BUG] Pure noise on inference with version 3.0.7

### ISSUE THREAD

**Issue by RebZJ** [IAHispano/Applio#307]
Issue

With this new update. On inference the output is pure giberrish, it's just noise.

Tested with 500 epochs, default params, all 3 samples rates, default pretrained model. 

I believe it has something to do with the model training, I tested with previous models that I have trained in earlier versions and it works fine for inference

Tested in both Paperspace and Colab.

**Comment by litsa-the-dancer** [comment]
Hm are u sure u cleaned up ur dataset properly, are u sure u loaded in the files as intended and most importantly would u mind sharing some screenshots of the parameters u used and how u go on about training a model

**Comment by RebZJ** [comment]
> Hm are u sure u cleaned up ur dataset properly, are u sure u loaded in the files as intended and most importantly would u mind sharing some screenshots of the parameters u used and how u go on about training a model

Hi, yes. This is a dataset that I've actually used previously. I just added a few more points and I also tested by removing those points. 

The parameters are default. Not sure if I need to share any screenshot for this. 

I just tested this dataset with the RVC web UI and it works for me.

**Comment by litsa-the-dancer** [comment]
Thing is, did u actually load in everything properly.

**Comment by RebZJ** [comment]
Hmm, I'm not sure how much more I can elaborate.

I've loaded it multiple ways. Creating the datasets folder manually and also using the dataset loader (copying my audio files in). 

Everything else is just the default parameters. I have preprocessed, extracted pitch, trained model, trained index. All the parameters are just the default ones. 

Steps were followed as suggested here: https://docs.applio.org/get-started/training/
